cmd/bootstrap/cmd/finalize.go

```diff
@@ -127,6 +125,9 @@ func finalize(cmd *cobra.Command, args []string) {
 	log.Info().Str("seed", hex.EncodeToString(flagBootstrapRandomSeed)).Msg("deterministic bootstrapping random seed")
 	log.Info().Msg("")
 
+	// TODO: we already have all nodes generated from rootblock step, instead of assembling them again
+	//  we just need to read them. Fix this.
```

Comment:
The method naming might be a bit misleading here:
* https://github.com/onflow/flow-go/blob/567811b09b688314d33c8cb24040b16f07f5af9e/cmd/bootstrap/cmd/finalize.go#L132
* https://github.com/onflow/flow-go/blob/567811b09b688314d33c8cb24040b16f07f5af9e/cmd/bootstrap/cmd/finalize.go#L136

actually _read_ the data from file and don't re-create it.

Reply:
renamed methods for reading node infos to reflect that fact that we are _reading_ the infos and not generating them: 
https://github.com/onflow/flow-go/pull/1371/commits/175116b76087943ec719fe04e9e4c3e11d963424